pyccel/ast/numpyext.py

```diff
@@ -786,22 +786,22 @@ def __new__(cls, a, dtype=None, order='K', subok=True, shape=None):
 class NumpyNorm(PyccelInternalFunction):
     """ Represents call to numpy.norm"""
     _dtype = NativeReal()
+    _shape = ()
+    _rank  = 0
```

Comment:
Where is the precision set? I can't find it but I'm not sure if that's just because it's hard to read between the open conversations